h2o-py/tests/testdir_apis/H2OAssembly/pyunit_h2oassembly_divide.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../")
+from tests import pyunit_utils
+from h2o.assembly import *
+from h2o.utils.typechecks import assert_is_type
+
+def h2oassembly_divide():
```

Comment:
seems like tests for 
    divide = H2OFrame.__truediv__
    plus = H2OFrame.__add__
    multiply = H2OFrame.__mul__
    minus = H2OFrame.__sub__
    less_than = H2OFrame.__lt__
    less_than_equal = H2OFrame.__le__
    equal_equal = H2OFrame.__eq__
    not_equal = H2OFrame.__ne__
    greater_than = H2OFrame.__gt__
    greater_than_equal = H2OFrame.__ge__

can all be merged into a single testcase